plugins/dell-dock/README.md

```diff
@@ -71,6 +72,7 @@ This plugin uses the following plugin-specific quirks:
 | `DellDockBlobVersionOffset`  | The offset of the ASCII representation of a version string in a payload | 1.1.3                 |
 | `DellDockBoardMin`           | The minimum board revision required to safely operate the plugin        | 1.1.3                 |
 | `DellDockVersionLowest`      | The minimum component version required to safely operate the plugin     | 1.1.3                 |
+| `DellDockUSB4VersionLowest`  | The minimum component version required to use USB4 module               | 1.6.0                 |
```

Comment:
How to manage this if we backport to older branch e.g. 1.3.X?

Reply:
>How to manage this if we backport to older branch e.g. 1.3.X?

This information is most useful for building a CAB and what version you put as minimum in the XML.  I think when we backport we can adjust it as far back as we want, but the thing to keep in mind is that all the "in between" versions won't be able to handle it.  So if you put 1.3.13 then 1.4.x, 1.5.x and 1.6.0 really **won't** be able to handle it.

Reply:
Thanks for the explanation, in order to support early branch tool I think XML will keep it open not saying a specific version.